src/System.Security.Cryptography.Encryption/System.Security.Cryptography.Encryption.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.22911.2
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Security.Cryptography.Encryption", "src\System.Security.Cryptography.Encryption.csproj", "{D04A73AE-E418-4ACD-A132-7688435BE8B5}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Security.Cryptography.Encryption.Tests", "tests\System.Security.Cryptography.Encryption.Tests.csproj", "{54C20799-0F9C-4921-81E0-B3BEB73B451A}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{D04A73AE-E418-4ACD-A132-7688435BE8B5}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
```

Comment:
Can we add configs for the other OSes, too? e.g. https://github.com/dotnet/corefx/blob/master/src/System.Console/System.Console.sln#L26-L49